spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/TaskAppDeploymentRequestCreator.java

```diff
@@ -108,7 +108,7 @@ public AppDeploymentRequest createRequest(
 				appDeploymentProperties, this.whitelistProperties);
 
 		List<String> updatedCmdLineArgs = (taskExecutionInformation.isComposed())?this.updateCommandLineArgs(commandLineArgs,
-				taskExecution, platformName, taskExecutionInformation.getTaskDefinition().getTaskName()):this.updateCommandLineArgs(commandLineArgs,
+				taskExecution, platformName, taskExecutionInformation.getTaskDefinition().getRegisteredAppName()):this.updateCommandLineArgs(commandLineArgs,
```

Comment:
`registeredAppName` is already available as a variable in this method.

Reply:
Done